### DISCUSSION TITLE
bBecome a contributor

### DISCUSSION THREAD

**Discussion by solomedia-account** [kedacore/keda#5690] [Q&A / Need Help] [answered]
Hi, i'm a system administrator who uses kubernetes alot to develop and deploy distributed Software systems. It'll mean alot to be a contributor to KEDA for expanding my knowledge about events driven architecture in Kubernetes.

**Comment by zroubalik** [comment, marked as answer]
Welcome @solomedia-account just pick an issue (code, documentation, testing...) you would like to start with and contribute. we are happy to have you on board.

See contributind guide:
https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md
